webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -256,9 +256,10 @@ public function viewAction(Request $request, int $submitId)
             ->join('s.problem', 'p')
             ->join('s.language', 'l')
             ->join('s.contest', 'c')
+            ->join('s.files', 'f')
             ->leftJoin('s.external_judgements', 'ej', Join::WITH, 'ej.valid = 1')
             ->leftJoin('s.contest_problem', 'cp')
-            ->select('s', 't', 'p', 'l', 'c', 'cp', 'ej')
+            ->select('s', 't', 'p', 'l', 'c', 'f', 'cp', 'ej')
```

Comment:
Does this fetch all file contents? Maybe just fetch the names?

Reply:
Good catch.